application/controllers/collection_instrument.py

```diff
@@ -76,9 +77,9 @@ def get_instrument_by_search_string(self, search_string=None, limit=None, sessio
         return result
 
     @with_db_session
-    def upload_to_bucket(self, exercise_id, file, ru_ref=None, classifiers=None, session=None):
+    def upload_seft_to_bucket(self, exercise_id, file, ru_ref=None, classifiers=None, session=None):
```

Comment:
Me being explicit again, this is clearly SEFT by the instrument it is setting, it's name should reflect that